src/instructlab/model/backends/backends.py

```diff
@@ -172,15 +157,14 @@ def get(logger: logging.Logger, model_path: pathlib.Path, backend: str) -> str:
     # When the backend is not set using the --backend flag, determine the backend automatically
     # 'backend' is optional so we still check for None or empty string in case 'config.yaml' hasn't
     # been updated via 'ilab config init'
-    if backend is None or backend == "":
+    if backend is None:
         logger.debug(
             f"Backend is not set using auto-detected value: {auto_detected_backend}"
         )
         backend = auto_detected_backend
     # If the backend was set using the --backend flag, validate it.
     else:
         logger.debug(f"Validating '{backend}' backend")
-        validate_backend(backend)
```

Comment:
I'm keeping the else since https://github.com/instructlab/instructlab/pull/1576 will need it. Ideally this should merge prior to https://github.com/instructlab/instructlab/pull/1576.